src/main/java/seedu/address/logic/commands/AddExerciseCommand.java

```diff
@@ -24,13 +24,12 @@
     public static final String COMMAND_WORD = "add-e";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an exercise done by a client in FitBiz. "
-        + "Parameters: INDEX (must be a positive integer) "
```

Comment:
Why is the `"Parameters: "` also removed?

Reply:
oof fkup